tests/permissions/test_set_permission.py

```diff
@@ -34,3 +34,53 @@ async def test_permissions_set_permission(websocket, context_id, example_url):
     assert resp == {}
     assert await query_permission(websocket, context_id,
                                   'geolocation') == 'prompt'
+
+
+@pytest.mark.asyncio
+@pytest.mark.skip(reason="See chromium-bidi/issues#1610")
+async def test_permissions_set_permission_in_user_context(
+        websocket, context_id, example_url):
+    await goto_url(websocket, context_id, example_url)
+
+    user_context = await execute_command(websocket, {
+        "method": "browser.createUserContext",
+        "params": {}
+    })
+
+    browsing_context = await execute_command(
+        websocket, {
+            "method": "browsingContext.create",
+            "params": {
+                "type": "tab",
+                "userContext": user_context["userContext"]
+            }
+        })
```

Comment:
NIT: you can use `create_context` fixture and call it with `user_context["userContext"]` argument.